orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationTemplate.java

```diff
@@ -41,10 +42,18 @@
 @NoArgsConstructor(force = true, access = AccessLevel.PACKAGE)
 @Beta
 public class OrchestrationTemplate extends TemplateConfig {
+
+  /** Please use {@link #withTemplateMessages(List)} instead. */
   @JsonProperty("template")
   @Nullable
+  @With(onMethod_ = {@Deprecated})
   List<ChatMessage> template;
 
+  @JsonIgnore
+  @Nullable
+  @With(AccessLevel.PRIVATE)
+  List<Message> messages;
+
```

Comment:
(Major)

We don't need this field really, as long as we store the low lever object.
Similar to the other fields that we have.

```suggestion
```

Reply:
The getter would be missing, is this ok?